doc/schemas/app_openapi.json

```diff
@@ -7,16 +7,11 @@
             "$ref": "#/components/schemas/uint64"
           }
         },
-        "required": [
-          "caller_id"
-        ],
+        "required": ["caller_id"],
```

Comment:
These openapi files are generated so I think they should be ignored.

Reply:
Yes, the reformatting actually breaks the test.